backend/pkg/modules/README.md

```diff
@@ -0,0 +1,20 @@
+### Create a new module
```

Comment:
I think it will be wonderful to add a diagram here explaining the models/alerts/annotation architecture

Reply:
it's a good idea, we can add it later